tool/timer.re

```diff
@@ -6,6 +6,30 @@ struct TimerState {
     is_timer_active: bool,
 }
 
+fn save_splits(path: string) {
+    match Tas::timer_save_splits(path) {
+        Result::Ok(_unit) => return,
```

Comment:
Nit: This could be interpreted as a guard-if, even though it wants to be the exact opposite. It might make more sense to not early-return here and instead fall-through to the end of the function.